html/css/custom.css

```diff
@@ -40,14 +40,22 @@ p.release {
 
 .site-header {
   background-color: black;
-  background-image: url(../img/background13.jpg);
```

Comment:
If a browser doesn't support JS, then they won't have any background. We should keep the default.

Reply:
Make sense, only canging to background1, as I renamed them on a previous commit.